docker/run.sh

```diff
@@ -1,5 +1,17 @@
 #!/bin/bash
 
+tail_file() {
+  echo "tailing file $1"
+  ALIGN=27
+  LENGTH=`echo $1 | wc -c`
+  PADDING=`expr ${ALIGN} - ${LENGTH}`
+  PAD=`perl -e "print ' ' x $PADDING;"`
+  file="/var/log/$1"
+  # each tail runs in the background but prints to stdout
+  # sed outputs each line from tail prepended with the filename+padding
+  tail -qF $file | sed "s|^|$1${PAD}:|g" &
```

Comment:
```suggestion
  ALIGN=27
  PREFIX=$(printf "%-${ALIGN}s:" "$1")
  file="/var/log/$1"
  # each tail runs in the background but prints to stdout
  # sed outputs each line from tail prepended with the filename+padding
  tail -qF $file | sed --unbuffered "s|^|$PREFIX|g" &
```

This adds "--unbuffered" to sed because "docker logs" was not showing updates until several new lines appeared in the file. Now it streams updated lines immediately.

I also converted it to printf for simplicity and to avoid a call to perl.